### PR TITLE
Fix ESLint

### PR DESCRIPTION
**Pull request by felixfbecker**
ESLint wasn't even running in CI, that's why my previous PR passed 🤦‍♂ 
This fixes the remaining cases that weren't fixable with `--fix`.

### CODE REVIEW

File: extensions/go/src/xrefs.ts
Comment (felixfbecker) [inline extensions/go/src/xrefs.ts]: Caught a legit bug!

**Comment by felixfbecker** [comment]
@sourcegraph/code-intel ready for review

File: extensions/go/src/xrefs.ts
Comment (eseliger) [reply extensions/go/src/xrefs.ts]: robots take over the world!

**Review by eseliger — APPROVED** [review]

**Comment by efritz** [comment]
Please wait for a code owner approval before merging.

**Pull request merged**
